packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -19,6 +19,7 @@ export interface ISelectProps
   value: string[] | string | number;
   onChange: React.ChangeEventHandler<HTMLSelectElement>;
   ref?: React.ForwardedRef<HTMLSelectElement>;
+  ariaLabel: string;
```

Comment:
can you omit "aria-label" from the HTMLSelectElement props since we are passing it in via this prop?